jenkins/version-def.sh

```diff
@@ -49,8 +49,8 @@ echo "CUDF_VER: $CUDF_VER, CUDA_CLASSIFIER: $CUDA_CLASSIFIER, PROJECT_VER: $PROJ
     SPARK_VER: $SPARK_VER, SCALA_BINARY_VER: $SCALA_BINARY_VER"
 
 
-SPARK_SHIM_VERSIONS_STR=${SPARK_SHIM_VERSIONS_STR:-"311 321cdh 312 313 314 320 321 322 330 331"}
-
-IFS=" " <<< $SPARK_SHIM_VERSIONS_STR read -r -a SPARK_SHIM_VERSIONS
-
+SPARK_SHIM_VERSIONS_STR=$(mvn -B help:evaluate -q -pl dist -PnoSnapshots -Dexpression=included_buildvers -DforceStdout)
```

Comment:
this part may not always be `noSnapshots`.
regular phase: noSnapshots + snapshots
pre-release phase: noSnapshots only

so it would be nice to have some switch control that we could easy enable&disable during release period

Reply:
added an argument for switch control:

- regular -> `PnoSnapshots` (noSnapshots only)
- pre-release -> `snapshots` (noSnapshots + snapshots)
- \* -> `noSnapshots` (noSnapshots only)